less/category.less

```diff
@@ -130,13 +130,24 @@
 		color: @gray-light;
 	}
 
-	h2.title .description {
-		font-size: 65%;
+	.description {
+		font-size: 0.73125rem;
```

Comment:
Why did this change get added? It makes category descriptions really tiny, so we've reverted it now.

Reply:
I wanted to prevserve the size...
Unwanted side effect, sorry about that